install_config/advanced_ldap_configuration/sssd_for_ldap_failover.adoc

```diff
@@ -398,26 +379,22 @@ created. To do so:
 ====
 ----
   identityProviders:
-  - name: any_provider_name
+  - name: sssd
     challenge: true
-    login: false
+    login: true
     mappingMethod: claim
     provider:
       apiVersion: v1
-      kind: RequestHeaderIdentityProvider
-      challengeURL: "https://proxy.example.com/challenging-proxy/oauth/authorize?${query}"
-      clientCA: /etc/origin/proxy/proxyca.crt
-      headers:
-      - X-Remote-User
+      kind: BasicAuthPasswordIdentityProvider
+      url: https://remote-basic.example.com/check_user.php
+      ca: /etc/origin/master/ca.crt
+      certFile: /etc/origin/master/openshift-master.crt
+      keyFile: /etc/origin/master/openshift-master.key
 ----
 ====
 +
 [NOTE]
 ====
-xref:../advanced_ldap_configuration/configuring_form_based_authentication.adoc#configuring-form-based-authentication[Configuring
-Form-Based Authentication] explains how to add the login URL to support web
-logins.
-
 xref:../advanced_ldap_configuration/configuring_extended_ldap_attributes.adoc#configuring-extended-ldap-attributes[Configuring
```

Comment:
Realistically, would a user ever want to complete the steps in this topic without completing the steps in the next topic?